<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consulta_professor</class>
 <widget class="QWidget" name="consulta_professor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consulta</string>
  </property>
  <widget class="QLabel" name="label_variavel">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>68</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campo_pesquisa">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>64</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radio_cpf">
      <property name="text">
       <string>CPF</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_nome">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_pesquisar">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabela_resultado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>961</width>
     <height>431</height>
    </rect>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerItem</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::CustomDashLine</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radio_cpf</tabstop>
  <tabstop>radio_nome</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
